Sources/PortalSwift/Core/Mpc/PortalMpcTypes.swift

```diff
@@ -0,0 +1,154 @@
+public struct PortalMpcBackupResponse {
```

Comment:
Types from the `PortalMpc.swift` file have been moved to this file for organization purposes